src/current/v22.1/import-performance-best-practices.md

```diff
@@ -5,28 +5,89 @@ toc: true
+1. [`AVRO`](migrate-from-avro.html).
+
+We recommend formatting your import files as `CSV` or `AVRO`. These formats can be processed in parallel by all the nodes in the cluster, which increases performance. To import in these formats, use [`IMPORT INTO`](import-into.html).
+
+{% include {{ page.version.version }}/import-table-deprecate.md %}
+
+### Import the schema separately from the data
+
+When importing into a new table split your dump data into two files:
+
+1. A SQL file containing the table schema.
+1. A CSV, delimited, or AVRO file containing the table data.
+
+Convert the schema-only file using the [Schema Conversion Tool](../cockroachcloud/migrations-page.html). The Schema Conversion Tool automatically creates a new {{ site.data.products.serverless }} database with the converted schema. {% include cockroachcloud/migration/sct-self-hosted.md %}
+
+Then use the [`IMPORT INTO`](import-into.html) statement to import the CSV data into the newly created table:
+
+~~~ sql
+> IMPORT INTO customers (id, name)
+CSV DATA (
+    'https://s3-us-west-1.amazonaws.com/cockroachdb-movr/datasets/employees-db/pg_dump/customers.csv'
+);
+~~~
+
+This method has the added benefit of alerting on potential issues with the import sooner; that is, you will not have to wait for the file to load both the schema and data just to find an error in the schema.
+
+### Import into a schema with secondary indexes
+
+When importing data into a table with secondary indexes, the import job will ingest the table data and required secondary index data concurrently. This may result in a longer import time compared to a table without secondary indexes. However, this typically adds less time to the initial import than following it with a separate pass to add the indexes. As a result, importing tables with their secondary indexes is the default workflow, an effective strategy for most migrations.
+
+However, in **large** imports (that is, datasets larger than 100 GiB in total size), it may be preferable to temporarily remove the secondary indexes from the schema, perform the import, and then re-create the indexes separately. Write operations on tables with many secondary indexes take longer to complete, and importing large datasets have a greater risk of timeouts. Removing the table's secondary indexes allows you to separate the initial data import from the secondary index creation operation, and the total import time is lower. Separating these operations also provides increased visibility into each operation's progress, and the ability to retry each operation independently if you encounter errors or timeouts.
+
+When recreating the secondary indexes, execute the [`CREATE INDEX`](create-index.html) statements one at a time for best performance. For example:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE INDEX idx1 ON table1;
+CREATE INDEX idx2 ON table1;
+CREATE INDEX idx3 ON table1;
+~~~
+
+- [Remove the secondary indexes](drop-index.html)
```

Comment:
Should these three bullets' links just be integrated into the "However, in large imports..." paragraph? 

If you want to keep these as bullets, I think they would benefit from a intro sentence before them.